chakracore/8/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM buildpack-deps:jessie
+FROM buildpack-deps:stretch
```

Comment:
I don't think we wanna change this in a security release

Reply:
Seems to be a side-effect of the update tool, and yes, I agree. I'll manually back this change out once I do the final updates.